Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Update.ts

```diff
@@ -295,17 +295,17 @@ async function updateDocumentByDocumentUuidTransaction(
       return invalidReferenceResult;
     }
   }
-  const document: MeadowlarkDocument = meadowlarkDocumentFrom(
+  const document: MeadowlarkDocument = meadowlarkDocumentFrom({
     resourceInfo,
     documentInfo,
     documentUuid,
     meadowlarkId,
     edfiDoc,
-    validateDocumentReferencesExist,
-    security.clientId,
-    Date.now(),
-    lastModifiedAt,
-  );
+    validate: validateDocumentReferencesExist,
+    createdBy: security.clientId,
+    createdAt: null,
```

Comment:
Restoring this to the actual created at: is that what you intended with the comment in an earlier file, about fixing the `createdAt` when replacing a document?

Reply:
Yes, clearly that is the case. I got myself lost in the thread while bouncing between this review and something else.